sql/telemetry_derived/clients_histogram_aggregates_merged_v1/init.sql

```diff
@@ -31,5 +31,5 @@ CREATE TABLE IF NOT EXISTS
       aggregates ARRAY<STRUCT<key STRING, value INT64>>
     >>,
 )
-PARTITION BY RANGE_BUCKET(channel_enum, GENERATE_ARRAY(0, 2, 1))
-CLUSTER BY app_version, client_id
+PARTITION BY RANGE_BUCKET(sample_id, GENERATE_ARRAY(0, 100, 1))
+CLUSTER BY app_version, channel, client_id
```

Comment:
Can partitioning be changed after the fact?

Reply:
Not in a simple way. I create a new table with the new partitioning scheme, populate it with the existing data from the old table + the new column, then rename it